tests/cpp-empty-test/proj.android/project.properties

```diff
@@ -8,6 +8,6 @@
 # project structure.
 
 # Project target.
-target=android-10
+target=android-22
```

Comment:
It is automatically generated because I have only api 22 installed on my machine. I will revert that back.